hbt/calibration/default.py

```diff
@@ -153,14 +156,15 @@ def add_calib_cls(name, base, cls_dict=None):
             "met_name": met_name,
             "jec_uncertainty_sources": [],
         })
-        # derive tec calibrators
-        add_calib_cls("tec_full", tec, cls_dict={
-            "propagate_met": False,  # not needed after JET-to-MET propagation
-        })
-        add_calib_cls("tec_nominal", tec, cls_dict={
-            "propagate_met": False,  # not needed after JET-to-MET propagation
-            "with_uncertainties": False,
-        })
+        if self.config_inst.campaign.x.year!=2024:
```

Comment:
```suggestion
        # TODO: remove if statement wenn tec corrections are made available
        if self.config_inst.campaign.x.year != 2024:
```

add TODO to not forget to change it as soon as available